src/main/resources/static/static/js/updateOrder.js

```diff
@@ -0,0 +1,68 @@
+//计算商品总价
+function updateFinalPrice() {
+    var $tr = $("#itemTable").find("tr[id]");
+    var summer = 0;
```

Comment:
Please do not name a variable starting with `$`.

Reply:
solved